kuma/core/management/commands/ihavepower.py

```diff
@@ -0,0 +1,28 @@
+from django.core.management.base import BaseCommand, CommandError
+
+from kuma.users.models import User
```

Comment:
Please use `django.contrib.auth.get_user_model` here instead to fetch the User model.